StRoot/StBFChain/StBFChain.cxx

```diff
@@ -654,6 +654,16 @@ Int_t StBFChain::Instantiate()
     // Use status tables for raw BEMC data (helpful for QA)
     if ( maker == "StEmcRawMaker" && GetOption("BEmcChkStat"))
       mk->SetAttr("BEmcCheckStatus",kTRUE);
+      
+    // 
+    if ( maker == "StEmcRawMaker" && GetOption("picoWrite")) {
+      mk->SetMode(1);  // saveAllStEvent for picoDst
```

Comment:
Just documenting here, since I didn't know about this before, that this mode enables some additional hit-saving in StEmcRawMaker.cxx:

    if((m_Mode&0x1)==1)
    {
                LOG_INFO << "Setting BEMC debug Mode -> save all hits into StEvent"<<endm;
        mBemcRaw->saveAllStEvent(kTRUE);
        mBemcRaw->initQAHisto();
    }

-Gene